### PR TITLE
Add script to count unreleased commits + cleanup

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: bin/create-pull-requests [hunk 1]
```diff
@@ -7,7 +7,7 @@ fi
 
 version=$(git describe)
 for module in modules/* ; do
-  pushd $module
+  pushd "$module" > /dev/null || exit
```
Comment (ekohl) [inline bin/create-pull-requests]: How about ```#!/bin/bash -e``` then?

File: bin/count-unreleased-commits [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+for module in modules/* ; do
+  pushd "$module"  > /dev/null || exit
```
Comment (ekohl) [inline bin/count-unreleased-commits]: Rather than using pushd you could also use ```git --git-dir``` I think.

Comment (bastelfreak) [reply bin/count-unreleased-commits]: fixed

File: bin/create-pull-requests (hunk 1 above)
Comment (bastelfreak) [reply bin/create-pull-requests]: done

Comment (ekohl) [reply bin/create-pull-requests]: You can remove the || exit since that's what ```-e``` does.

File: bin/count-unreleased-commits [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+for module in modules/* ; do
+  #pushd "$module"  > /dev/null || exit
```
Comment (ekohl) [inline bin/count-unreleased-commits]: Can you remove the comments?

File: bin/create-pull-requests (hunk 1 above)
Comment (bastelfreak) [reply bin/create-pull-requests]: done

File: bin/count-unreleased-commits (hunk 3 above)
Comment (bastelfreak) [reply bin/count-unreleased-commits]: done

**Pull request merged**